### PR TITLE
Develop to master

### PR DESCRIPTION
**Pull request by matinlotfali**

### CODE REVIEW

File: content/HowToSetUpSocketCANInterface.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
----
-title: "CAN setup"
----
+# CAN setup
 # How To Set Up Socket CAN Interface
```
Comment (mistyron) [inline content/HowToSetUpSocketCANInterface.md:2]: Two H1 headers behind each other might look a bit strange, wouldn't it?

**Review by mistyron — APPROVED** [review]
Madew a comment about HowToSetUpSocketCANInterface.md but generally, LGTM :+1:

File: content/SetupWifi.md [hunk 2]
```diff
@@ -1,6 +1,4 @@
----
-title: "Setup WiFi"
----
+# Setup WiFi
 
 # Setup WiFi
```
Comment (mistyron) [inline content/SetupWifi.md:3]: Two H1 titles

File: content/StartingSSH.md [hunk 3]
```diff
@@ -1,6 +1,4 @@
----
-title: "Starting SSH"
----
+# Starting SSH
 
 # SSH Server
```
Comment (mistyron) [inline content/StartingSSH.md:3]: Two H1 titlesTwo H1 titles

File: content/Using_GPIOs.md [hunk 4]
```diff
@@ -1,6 +1,4 @@
----
-title: "Using GPIOs"
----
+# Using GPIOs
 
 # Using GPIO on MistySOM and MistyCarrier to Test Toggling a LED With Libgpiod Utilities
```
Comment (mistyron) [inline content/Using_GPIOs.md:3]: Two H1 titles

**Comment by matinlotfali** [comment]
Nice catch. Fixed the titles and I am merging to master

**Pull request merged**